spec.bs

```diff
@@ -8037,20 +8170,49 @@ into smaller number of fetches. It's a [=struct=] with the following [=struct/it
   : <dfn>length limit</dfn>
   :: A {{long}}, initially 2147483647 (the maximum value that it can hold). Describes the URL length
      limit the current batch is limited to, the smallest of the limits of the interest groups included.
+  : <dfn>key value interest groups</dfn>
+  :: An [=ordered map=] whose [=map/keys=] are [=tuples=] of (an [=origin=] for
+    [=interest group/joining origin=], an [=origin=] for [=interest group/owner=]) and whose
+    [=map/values=] are [=sets=] of [=interest groups=].
 </dl>
 
 A <dfn>bidding signals per interest group data</dfn> is a [=struct=] with the following
 [=struct/items=]:
 
 <dl dfn-for="bidding signals per interest group data">
-  : <dfn>updateIfOlderThanMs</dfn>
+  : <dfn>update if older than ms</dfn>
   :: Null or a {{double}}. If non-null, it is the [=duration=] in milliseconds after which the
     [=interest group=] is eligible for an [update](#interest-group-update), if it hasn't been joined
     or updated within that duration. When eligible, the update is performed either following a call
     to {{Navigator/runAdAuction()}}, for each of the passed {{AuctionAdConfig/interestGroupBuyers}},
     or explicitly by the {{Navigator/updateAdInterestGroups()}} method.
 </dl>
 
+A <dfn>compression group</dfn> is a collection of [=partitions=] that can be compressed together
+in the signals response. It's a [=struct=] with the following [=struct/items=]:
+
+<dl dfn-for="compression group">
+  : <dfn>compression group id</dfn>
+  :: An integer indicates the index of this compression group.
+  : <dfn>partitions</dfn>
+  :: A [=list=] of [=partition=]. Contains all the partitions belong to this compression group.

```

Comment:
```suggestion
  :: A [=list=] of [=partition=]. Contains all the [=partitions=] belong to this compression group.
```

Reply:
Done, but curious why we want the reference to the`partitions` under its own description?